default.nix

```diff
@@ -4,13 +4,16 @@
 
 let
   xxd = pkgs.callPackage ./nix/xxd {};
+  fulltoc = pkgs.callPackage ./nix/sphinx-fulltoc {};
 
 in with pkgs; {
   coreEnv = stdenv.mkDerivation {
     name = "thecore";
     buildInputs = [ python35Packages.cogapp which cmake gcc6 gdb cppcheck
                     cpputest gcc-arm-embedded-5 dfu-util
-                    doxygen clang openocd perlPackages.ArchiveZip xxd ];
+                    doxygen clang openocd perlPackages.ArchiveZip xxd
+                    python27Packages.sphinx python27Packages.sphinx_rtd_theme
```

Comment:
Hm, probably I missed something when performing package search.
Do not want to leave this PR opened. Let's move it to #276